jenkins/bootstrap_test.py

```diff
@@ -1825,25 +1833,27 @@ def CheckBootstrapYaml(self, path, check, use_json=False):
                     self.fail('Jobs may not contain child objects %s: %s' % (
                         key, value))
                 if '{' in str(value):
-                    self.fail('Jobs may not contain {expansions}' % (
+                    self.fail('Jobs may not contain {expansions} - %s: %s' % (
                         key, value))  # Use simple strings
             # Things to check on specific flavors.
             job_name = check(real_job, name)
             self.assertTrue(job_name)
             self.assertEquals(job_name, real_job.get('job-name'))
 
-    def GetRealBootstrapJob(self, job):
+    def get_real_bootstrap_job(self, job):
         key = os.path.splitext(job.strip())[0]
         if not key in self.realjobs:
-            for yaml in self.yaml_suffix:
-                self.LoadBootstrapYaml(yaml)
-            self.LoadProwYaml(self.prow_config)
+            for yamlf in self.yaml_suffix:
+                self.load_bootstrap_yaml(yamlf)
+            self.load_prow_yaml(self.prow_config)
         self.assertIn(key, sorted(self.realjobs))  # sorted for clearer error message
         return self.realjobs.get(key)
 
-    def testValidTimeout(self):
+    def test_valid_timeout(self):
         """All jobs set a timeout less than 120m or set DOCKER_TIMEOUT."""
-        default_timeout = int(re.search(r'\$\{DOCKER_TIMEOUT:-(\d+)m', open('%s/dockerized-e2e-runner.sh' % os.path.dirname(__file__)).read()).group(1))
+        default_timeout = int(re.search(r'\$\{DOCKER_TIMEOUT:-(\d+)m',
```

Comment:
Please break the the line after a ( (ether int( or re.search( so that each args indent doesnt' depend on the length of the previous line